### PR TITLE
Severity vocabulary

### PR DESCRIPTION
**Pull request by ereteog**
Dedicated Severity Vocabulary.

### CODE REVIEW

**Review by msprunck — APPROVED** [review]
LGTM!

**Review by rplevy — APPROVED** [review]
LGTM!

**Review by agzam — APPROVED** [review]
LGTM! :thumbsup:

File: src/ctim/schemas/vocabularies.cljc [hunk 1]
```diff
@@ -84,6 +83,18 @@
   :reference (str "[HighMedLowVocab](http://stixproject.github.io/"
                   "data-model/1.2/stixVocabs/HighMediumLowVocab-1.0/)"))
 
+(def severity
+  #{"Info"
```
Comment (ereteog) [inline src/ctim/schemas/vocabularies.cljc:87]: Severity Info is used in production.

**Pull request merged**